VAMobile/src/utils/remoteConfig.test.ts

```diff
@@ -26,6 +26,8 @@ const mockOverrides = {
   haptics: false,
   whatsNewUI: false,
   decisionLettersWaygate: false,
```

Comment:
What are the mockOverrides? For unit and/or detox tests? Looks like some of these are false on released features which seems odd.